applications/acdc/src/acdc_agent_fsm.erl

```diff
@@ -561,8 +555,8 @@ ready({'member_connect_win', JObj}, #state{agent_listener=AgentListener
             case get_endpoints(OrigEPs, AgentListener, Call, AgentId, QueueId) of
                 {'error', 'no_endpoints'} ->
                     lager:info("agent ~s has no endpoints assigned; logging agent out", [AgentId]),
-                    acdc_agent_listener:logout_agent(AgentListener),
                     acdc_agent_stats:agent_logged_out(AccountId, AgentId),
+                    ?MODULE:agent_logout(self()),
```

Comment:
I may have been mindlessly replacing the old 'acdc_agent_listener' call as ?MODULE. I am happy to take it out as it is unnecessary